font/src/ft/mod.rs

```diff
@@ -89,8 +89,8 @@ fn to_freetype_26_6(f: f32) -> isize {
 }
 
 #[inline]
-fn to_fixedpoint_16_6(f: f64) -> i64 {
-    (f * 65536.0) as i64
+fn to_fixedpoint_16_6(f: f64) -> c_long {
+    (f * 65536.0) as c_long
```

Comment:
Can't we just use an `isize` here?

Reply:
Is it a thing that `clong` matches `isize` on any combination of arches + bitness? I'm not sure that it's true and it's not stated anywhere, so I'd like to avoid that, so I've picked `clong` for that to void any potential issues in any form.

Reply:
I assumed that was your reasoning, which makes sense, yeah. Matching the external interface is usually the better choice.

But would you mind changing `to_freetype_26_6` too then? Just for consistency.

Reply:
The other API is in isize.